edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/ConnectionProvider.cs

```diff
@@ -1,23 +1,29 @@
+// <copyright file="ConnectionProvider.cs" company="Microsoft">
 // Copyright (c) Microsoft. All rights reserved.
-
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Are these changes mandated now?

Reply:
Yes, file header is enforced in StyleCop. Do you like it?

Reply:
No, I don't...

Reply:
So you prefer not adding file header for all files? or you just don't like the current file header format?

Reply:
Actually, let me take it back. I like the header.. and we should have one. I would prefer one as simple as possible, but I am okay with what you have.